app/src/format.py

```diff
@@ -134,11 +134,15 @@ def build_accordions(
     remapped_citations = remap_citation_ids(subsections, raw_response)
 
     citations_html = ""
+    list_of_accordion_ids = []
     for document, cited_subsections in _group_by_document(remapped_citations).items():
         _accordion_id += 1
         citation_body = _build_citation_body(config, document, cited_subsections)
         formatted_citation_body = config.format_accordion_body(citation_body)
         citation_numbers = [citation.id for citation in cited_subsections]
+        list_of_accordion_ids.append(
+            {"citation_numbers": citation_numbers, "accordion_id": _accordion_id}
+        )
```

Comment:
Instead of a list, could this be a dictionary where the accordion_id was the key, and the values where citation_numbers?